README.md

```diff
@@ -21,6 +21,7 @@ PRs are welcome.
 Converts [OData V4](http://docs.oasis-open.org/odata/odata/v4.0/odata-v4.0-part1-protocol.html) queries to [DocumentDB SQL](https://azure.microsoft.com/en-us/documentation/articles/documentdb-sql-query/) queries. 
 
 ## Release Notes
+* 3.1.80 Added support for optimized case-insensitive string queries
```

Comment:
please mention that this is a breaking change and tolower and toupper will no longer perform what they are expecting to do.